src/patternfly/components/Nav/nav.scss

```diff
@@ -354,180 +252,105 @@
-      bottom: var(--pf-c-nav__list-link--after--Bottom);
-      left: var(--pf-c-nav__list-link--after--Left);
-      width: var(--pf-c-nav__list-link--after--Width);
-      height: var(--pf-c-nav__list-link--after--Height);
-      content: "";
-    }
+  &::before {
+    border-color: var(--pf-c-nav__link--before--BorderColor);
+    border-style: solid;
+    border-top-width: var(--pf-c-nav__link--before--BorderTopWidth);
+    border-bottom-width: var(--pf-c-nav__link--before--BorderBottomWidth);
   }
 
-  // Link
-  .pf-c-nav__link {
-    position: relative;
-    display: flex;
-    align-items: baseline;
-    padding:
-      var(--pf-c-nav__list-link--PaddingTop)
-      var(--pf-c-nav__list-link--PaddingRight)
-      var(--pf-c-nav__list-link--PaddingBottom)
-      var(--pf-c-nav__list-link--PaddingLeft);
-    font-weight: var(--pf-c-nav__list-link--FontWeight);
-    color: var(--pf-c-nav__list-link--Color);
-
-
-    // States
-    &.pf-m-hover,
-    &:hover {
-      --pf-c-nav__list-link--Color: var(--pf-c-nav__list-link--hover--Color);
-
-      &::after {
-        background-color: var(--pf-c-nav__list-link--hover--after--BackgroundColor);
-      }
-    }
-
-    &.pf-m-active,
-    &:active {
-      --pf-c-nav__list-link--FontWeight: var(--pf-c-nav__list-link--active--FontWeight);
-      --pf-c-nav__list-link--Color: var(--pf-c-nav__list-link--active--Color);
-
-      &::after {
-        background-color: var(--pf-c-nav__list-link--active--after--BackgroundColor);
-      }
-    }
-
-    &.pf-m-focus,
-    &:focus {
-      --pf-c-nav__list-link--FontWeight: var(--pf-c-nav__list-link--focus--FontWeight);
-      --pf-c-nav__list-link--Color: var(--pf-c-nav__list-link--focus--Color);
-
-      &::after {
-        background-color: var(--pf-c-nav__list-link--focus--after--BackgroundColor);
-      }
-    }
+  &::after {
+    border-color: var(--pf-c-nav__link--after--BorderColor);
+    border-style: solid;
+    border-bottom-width: var(--pf-c-nav__link--after--BorderBottomWidth);
+    border-left-width: var(--pf-c-nav__link--after--BorderLeftWidth);
   }
 
   // States
-  .pf-m-current.pf-c-nav__link,
-  .pf-m-current > .pf-c-nav__link {
-    --pf-c-nav__list-link--FontWeight: var(--pf-c-nav__list-link--m-current--FontWeight);
-    --pf-c-nav__list-link--Color: var(--pf-c-nav__list-link--m-current--Color);
-
-    &::after {
-      background-color: var(--pf-c-nav__list-link--m-current--after--BackgroundColor);
-    }
+  &:hover {
+    color: var(--pf-c-nav__link--hover--Color);
+    background-color: var(--pf-c-nav__link--hover--BackgroundColor);
   }
-}
-
 
-// Separator --- remove this block at breaking change
-.pf-c-nav__separator {
-  height: var(--pf-c-nav__separator--Height);
-  margin-top: var(--pf-c-nav__separator--MarginTop);
-  margin-bottom: var(--pf-c-nav__separator--MarginBottom);
-  margin-left: var(--pf-c-nav__separator--MarginLeft);
-  background-color: var(--pf-c-nav__separator--BackgroundColor);
+  &:active {
```

Comment:
Not part of the design, but can you move anywhere we're defining `:active` along with `:focus` so that the `:active` styles come after? Otherwise they'll never work, since `:focus` will always override.